src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../react-app-env';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<{ movies: Movie[]}> = ({
+  movies,
+}: { movies: Movie[] }) => (
+  <div className="movies">
+    {movies.map((mov: Movie) => (
+      <MovieCard {...mov} />
```

Comment:
Explicitly pass the parameters to the component without ...spread. Use longer notation - title={mov.title}, because it's more clear, what data you pass in component.